pkg/flexvolume/client.go

```diff
@@ -38,18 +60,33 @@ func (c *client) Detach(mountDevice string) error {
 			MountDevice: mountDevice,
 		},
 	)
+
+	output := newOutput(err)
+	if _, osErr := os.Stdout.Write(output); osErr != nil {
```

Comment:
if you really want to log here (not sure), then you can do it inside the `writeOutput` function too